bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/categorylist/FeaturedCategoryListImpl.java

```diff
@@ -120,7 +127,8 @@ private void initModel() {
         }
         List<CategoryTree> categories = categoriesRetriever.fetchCategories();
         for (CategoryTree category : categories) {
-            category.setPath(String.format("%s.%s.html", categoryPage.getPath(), category.getId()));
+            Map<String, String> params = new ParamsBuilder().id(category.getId().toString()).map();
```

Comment:
`CategoriesRetriever` is also fetching the `url_path` (see https://github.com/adobe/aem-core-cif-components/blob/master/bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/categorylist/CategoriesRetriever.java#L29-L33). So why not pass it to the url provider as well?

Reply:
Good point, I added that